maker/src/routes.rs

```diff
@@ -34,12 +35,13 @@ pub fn router(
 
     Router::new()
         .route("/", get(index))
-        .route("/api/newaddress", get(get_unused_address))
+        .route("/api/new-address", get(get_unused_address))
```

Comment:
Why the rename? In the coordinator it's `newaddress`

Reply:
I was just in the mood to make it more readable (and less typeable 😛), but I definitely don't have to change it.

Reply:
I have walked back the change. What I really wanted to rename was the `dlc_channels` endpoint to use kebab-case, but that's completely unrelated haha.